roles/lnls-ans-role-sirius-apps/vars/Debian-jessie.yml

```diff
@@ -8,10 +8,6 @@ __sirius_apps_packages:
   - "liblapack-dev={{ pkg_version_liblapack_dev }}"
   - "libgsl-dev={{ pkg_version_libgsl_dev }}"
 
-__sirius_apps_pip_packages:
-  - "pyvisa=={{ pkg_version_pyvisa }}"
-  - "pyvisa-py=={{ pkg_version_pyvisa_py }}"
```

Comment:
Does `pyvisa` and `pyvisa-py` are only used for the controls opi screens? @xresende , @fernandohds564 , @carneirofc ? Is this really safe to remove from the role?

Still, even if we remove this we should keep the as an empty list for documentation, as in: `__sirius_apps_pip_packages: []`

Reply:
We also use `pyvisa` and `pyvisa-py` in our Linac and transport line currinfo IOCs, @lerwys. I think we cannot remove them from this role, @carneirofc. Sorry for not spotting this on my review and thanks for the note, @lerwys.